onnxmltools/utils/float16_converter.py

```diff
@@ -17,16 +22,28 @@ def convert_float_to_float16(model):
     '''
     Convert TensorProto DataType float in the ONNX model to float16.
 
-    :param model: ONNX model
-    :return: converted ONNX model
+    :param model: ONNX model ProtoBuf object
+    :return: converted ONNX model ProtoBuf object
 
     Example:
 
     ::
 
+        Example 1: Convert ONNX model ProtoBuf object:
+        from onnxmltools.utils.float16_converter import convert_float_to_float16
+        new_onnx_model = convert_float_to_float16(onnx_model)
+
```

Comment:
This method is very intuitive, shall we just keep one example?

Reply:
The thinking is that for external user they may want to know how to use this method as a standalone feature to convert onnx binary file.

Reply:
Agree with @stevenlix . Examples are always not enough (until the examples we have can cover ALL user cases).